rasa/core/policies/sklearn_policy.py

```diff
@@ -338,7 +338,7 @@ def persist(self, path: Union[Text, Path]) -> None:
     @classmethod
     def load(
         cls, path: Union[Text, Path], should_finetune: bool = False, **kwargs: Any
-    ) -> Policy:
+    ) -> Optional[Policy]:
```

Comment:
we can't do this @joejuzl as this goes against the `Policy` interface

Reply:
haha yes true. `TEDPolicy.load` was recently changed to have `return cls()`. We can do that here too.

Reply:
yes please